etc/openqa/openqa.ini

```diff
@@ -149,6 +149,13 @@ blocklist = job_grab job_done
 #exchange = pubsub
 #topic_prefix = suse
 
+# Cleanup related configuration (besides limits)
+[cleanup]
+# Specifies whether different cleanup tasks can run in parallel. This makes
+# option particularly sense if a different storage for assets and results is
```

Comment:
s/if a different storage for assets and results is used/if assets and results are stored on different storage locations/

Also I would add "… or if your common storage solution is performant enough".